apis/src/providers/schedules.rs

```diff
@@ -0,0 +1,16 @@
+use std::collections::HashMap;
+
+use leptos::{provide_context, RwSignal};
+use shared_types::GameId;
+
+use crate::responses::ScheduleResponse;
+
+#[derive(Default, Clone)]
+pub struct SchedulesContext {
+    pub my_schedules: RwSignal<HashMap<GameId, Vec<ScheduleResponse>>>,
```

Comment:
What does splitting them into two help with?

Reply:
a logged off user should still be able to see all agreed schedules hence its safer to isolate the info sent to each client so it will only contain schedules relevant to them